packages/client/scripts/colors.js

```diff
@@ -3,8 +3,8 @@
 const isObject = (val) => val !== null && typeof val === 'object' && !Array.isArray(val)
 
 // this is a modified version of https://github.com/chalk/ansi-regex (MIT License)
-/* eslint-disable-next-line no-control-regex */
 const ANSI_REGEX =
+  // eslint-disable-next-line no-control-regex
```

Comment:
I thought I fixed that on main, is this branch up-to-date already?

Reply:
and same comments for other places as well

Reply:
Merged with main that's gone now